config/environment.rb

```diff
@@ -19,6 +19,7 @@
 require "sinatra/reloader" if development?
 
 require 'erb'
+require 'faker'
```

Comment:
If you take a look at line 19 of this file, they use a statement such as `if development?`. 

I suggest we use that for both the `faker` gem as well as for `pry`.

What do you think?